packages/switch/src/switch.module.css

```diff
@@ -17,36 +19,38 @@
     outline-offset: 2px;
   }
 
-  &[data-disabled] {
-    cursor: not-allowed;
-    background-color: var(--gds-sys-color-icon-interactive-disabled);
-  }
-
   &[data-state="checked"] {
-    background-color: var(--gds-sys-color-icon-interactive-active);
+    background-color: var(--gds-sys-color-icon-interactive-default);
+
+    &:hover {
+      background-color: var(--gds-sys-color-icon-interactive-active-hover);
+    }
   }
 
-  &[data-state="checked"][data-disabled] {
+  &[data-disabled],
+  &[data-disabled][data-state="checked"] {
```

Comment:
Do you need this `[data-state="checked"]` additional selector here? Looks like it should overwrite`[data-state="checked"]` CSS already since it's the last one

Reply:
Yep, if it's checked just &[data-disabled] is not enough